### DISCUSSION TITLE
Aqlvoy

### DISCUSSION THREAD

**Discussion by ELProductions** [iptv-org/community#805] [Q&A]
### Channel Name

Aqlvoy

### Channel ID

Aqlvoy.uz

### Website

https://aqlvoy.uz/index.php/ru/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/uz.m3u#L2-L3
